### DISCUSSION TITLE
Am really confused

### DISCUSSION THREAD

**Discussion by JurkoDev** [music-assistant/support#1375] [Q&A] [answered]
I want to play to ha music player I don't see how and am confused how do I use the universal group player

**Comment by OzGav** [comment]
Your question ins't clear. Have you got anything working with MA?

**Reply by JurkoDev** [reply]
so i dont have anything working with am (players) and i want to use some that i have but they didnt show up like a vlc telnet and some other stuff that is in ha as media player

**Reply by OzGav** [reply, marked as answer]
Only certain player types are supported. More will be added in the future. Current support is detailed here https://github.com/orgs/music-assistant/discussions/710#discussioncomment-3343044
